qa/workunits/rbd/rbd_support_module_recovery.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+set -ex
+
+POOL=rbd
+IMAGE_PREFIX=image
+MNT_DIR_PREFIX=mnt
+NUM_IMAGES=20
+RUN_TIME=3600
+
+rbd mirror pool enable ${POOL} image
+rbd mirror pool peer add ${POOL} dummy
+
+# Create images and schedule their mirror snapshots
+for ((i = 1; i <= ${NUM_IMAGES}; i++)); do
+    rbd create --size 1G ${POOL}/${IMAGE_PREFIX}$i
+    rbd mirror image enable ${POOL}/${IMAGE_PREFIX}$i snapshot
+    rbd mirror snapshot schedule add -p ${POOL} --image ${IMAGE_PREFIX}$i 1m
+done
+
+# Run fio workloads on images via kclient
+# Test the recovery of the rbd_support module and its scheduler from their
+# librbd client being blocklisted while a exclusive lock gets passed around
+# between their librbd client and a kclient trying to perform exclusive
+# operations on the same image.
+for ((i = 1; i <= ${NUM_IMAGES}; i++)); do
+    DEVS[$i]=$(sudo rbd device map ${POOL}/${IMAGE_PREFIX}$i)
+    sudo fio --name=fiotest --filename=${DEVS[$i]} --rw=randrw --bs=4K \
+        --direct=1 --ioengine=libaio --size=800M --iodepth=2 \
```

Comment:
Given `--runtime=43200 --time_based`, `--size` shouldn't be needed.  It only leads to questions: what is this for, why is it 800M and not 1G (the size of the image), etc.